integreat_cms/cms/models/pages/page_translation.py

```diff
@@ -94,24 +94,51 @@ def short_url(self):
             "public:expand_page_translation_id", kwargs={"short_url_id": self.id}
         )
 
+    @cached_property
+    def mirrored_translation_text(self):
+        """
+        This method returns the text of the mirrored translation.
+        If there is no mirrored translation, it returns an empty string.
+        If there is a mirrored page but no translation, a html error message will be returned with languages that are translated as options.
+
+        :return: The text, as specified
+        :rtype: str
+        """
+        if not self.page.mirrored_page:
+            return ""
+
+        text = self.page.get_mirrored_page_translation(self.language.slug)
+        if text and text.content:
+            return text.content
+
+        error_message = (
+            self.language.message_partial_live_content_not_available
+            if self.content
+            else self.language.message_live_content_not_available
+        )
+        choices = (
+            self.page.mirrored_page.prefetched_major_public_translations_by_language_slug
+        )
+        error_message += "<br>" + "<br>".join(
+            f"&emsp;* <a href={settings.WEBAPP_URL + choice.get_absolute_url()}>{choice.language.native_name}</a>"
+            for choice in choices.values()
+        )
+        return f"<p>{error_message}</p>"
```

Comment:
```suggestion
        # Get all translations of this page which have a corresponding translation of the mirrored page
        translations = filter(
            None,
            [
                self.page.get_public_translation(language_slug)
                for language_slug in self.page.mirrored_page.prefetched_major_public_translations_by_language_slug.keys()
            ],
        )
        # Create bullet points for each available translation
        choices = "</li><li>".join(
            f"<a href={settings.WEBAPP_URL + translation.get_absolute_url()}>{translation.language.native_name}</a>"
            for translation in translations
        )
        return f"<p>{error_message}</p><ul><li>{choices}</li></ul>"
```
Two ideas:
1. I think it would be better to link to the translations of the current page instead of the mirrored page. Also, we should probably only list options where both translations exist - those of the current page and those of the mirrored page.
2. I think bullet points make more sense in this case. Sure, they don't look good in the preview at the moment, but this is a general problem (see issue  #496) and isn't necessarily part of this PR.